ruby/lib/ThreatExchange/http_methods.rb

```diff
@@ -0,0 +1,34 @@
+require "rest-client"
+
+module ThreatExchange
+  class Client
+    module HttpMethods
+      # Common GET Options:
+      #   limit - Defines the maximum size of a page of results. The maximum is 1,000.
+      #   since - Returns malware collected after a timestamp
+      #   until - Returns malware collected before a timestamp
+      #   fields - A list of fields to return in the response
+      #   sort_order - A given SortOrderType
```

Comment:
To avoid duplication in each of the methods I just pulled out these common params?